### DISCUSSION TITLE
Anyone using @urql/vue with Pinia for state management?

### DISCUSSION THREAD

**Discussion by bartenra** [urql-graphql/urql#2319] [Q&A]
I wonder if anyone is using urql with [Pinia](https://pinia.vuejs.org/) the new state management library that is replacing Vuex.

Do you use urql inside actions? Do you pass server data to it via actions? Or do you merge state from Pinia and urql inside components?

**Comment by farin** [comment]
I prefer actions

**Reply by vin-ni** [reply]
Could you share an example?
